basics.js

```diff
@@ -4,19 +4,22 @@
 
 var pets = ['cat', 'dog', 'wombat'];
 
+console.log(pets.each)
 ///// Question 2
 // Change my friend's hair color to "green", without modifying the line below.
 
 var friend = { name: 'Chris', age: 27, hairColor: 'blue' };
+this.hairColor = 'green'
 
 ///// Question 3
 // The function below is problematic. In 1-2 sentences explain why, and then fix the code.
 
 var squarePlusOne = function(n){
-  squared = n * n;
+  {squared = n * n;}
```

Comment:
Putting squared inside of an object doesn't make it not global, and I don't think that syntax would work.